paasta_tools/utils.py

```diff
@@ -2767,8 +2771,63 @@ def get_services_for_cluster(
     return instance_list
 
 
-def parse_yaml_file(yaml_file: str) -> Any:
-    return yaml.safe_load(open(yaml_file))
+def load_service_instance_configs(
```

Comment:
now that this exists I wonder if we can deprecate PaastaServiceConfigLoader, which seems like an overly OOP version of basically this function.

Reply:
I'll follow up on this (PAASTA-16358) - will take some work to make sure that the new function replaces its functionality everywhere it is used.

Reply:
yeah definitely low-priority